lib/components/user/stacked-pane-display.tsx

```diff
@@ -59,7 +59,8 @@ const StackedPaneDisplay = ({
         okayButton={{
           disabled: buttonClicked === 'okay',
```

Comment:
Also, disabling the button prevents the form from being submitted.

Reply:
Yep this was fixed